.github/workflows/close_trivial_pr.yaml

```
@@ -0,0 +1,23 @@
+name: Close Trivial README PRs
+on:
+  pull_request:
```

This would need to be 
```suggestion
  pull_request_target:
```

To perform the actions you want. You can also filter to only include readme changes.

**However**, be careful of script injections on that environment, like the other review comment noted